bf2fc6cc711aee1a0c2a/kas-fleet-manager
# fix(db): honors enable-db-debug flag when set to true

## Description


The custom logger didn't print the sql statements which are useful during debugging. I ran into this
issue when testing something locally. Let's honor this flag by using the default logger that comes
with gorm to print the SQL statements.

## Verification Steps


To verify the changes:

- 1. Before this patch, build the binary and run `./kas-fleet-manager
serve --enable-db-debug=true` this wouldn't print the SQL statements
- 2. Apply this patch locally, built the binary and run
`./kas-fleet-manager serve --enable-db-debug=true` and observe the
printed SQL statements
- 3. Disabe db debug by run `./kas-fleet-manager serve --enable-db-debug=false` again with code changes on this patch and observe that no SQL statements are printed out

## Checklist (Definition of Done)

- ~~[ ] Documentation added for the feature~~ There will be a troubleshooting guide and the usage of this flag might go there
